### DISCUSSION TITLE
Is gossip shared with peers, my node has no channel with?

### DISCUSSION THREAD

**Discussion by DerEwige** [ACINQ/eclair#2603] [Node operators]
When I connect to another node, some nodes keep the connection alive, even when no channel is opened.
If I have such a peer, that my node is connected for a while, but has no channel.
Will my node send gossip to this channel and receive gossip from this channel?

**Comment by t-bast** [comment]
First of all, it's important to understand that there are two types of gossip:

- gossip queries: explicit requests to fetch announcements and updates
- staggered broadcast: recent announcements and updates that were just received and will be automatically forwarded to peers in batches every 30 seconds

Your node will automatically send and receive staggered broadcast with all peers, even if you don't have a channel with them.

Your node will also automatically respond to gossip queries from your peers, even if you don't have a channel with them.
But your node will need send gossip queries to peers you don't have a channel with: but it will automatically do it once the first channel is opened.

**Reply by DerEwige** [reply]
> * staggered broadcast: recent announcements and updates that were just received and will be automatically forwarded to peers in batches every 30 seconds
> 
> Your node will automatically send and receive staggered broadcast with all peers, even if you don't have a channel with them.

I was mainly interested in the staggered broadcast case.
Thanks for confirming, that this is shared with all peers.
